src/Dropdown/Dropdown.js

```diff
@@ -485,6 +484,7 @@ Dropdown.propTypes = {
   // Transition lengths
   transitionEnterTimeout: PropTypes.number,
   transitionLeaveTimeout: PropTypes.number,
+  trigger: PropTypes.element,
```

Comment:
And could check for instance of `DropdownTrigger` right here if follow the approach suggested above.

Reply:
looks like we dont have an easy way checking if `trigger` is an instance of `DropdownTrigger`. /cc @orlandohohmeier

Reply:
We need to implement a custom validation similar to https://github.com/mesosphere/marathon-ui/commit/8ecd621c1d3ea4f2ec9e516d8823d1b46467bde5#diff-9ed932ecbb81eee77651242eed89601bR250  @nLight could you please file an issue to introduce such a util and it to the foundation backlog?

Reply:
Didn't `PropTypes.instanceOf` work? Or does it check only provided type and fails to check children?

Reply:
`PropTypes.instanceOf` doesn't work for components as components aren't instance of the perspective class but react element  thus we need to introduce a custom validator.